src/_posts/databases/postgresql/2000-01-01-managing.md

```diff
@@ -161,6 +161,51 @@ ALTER DEFAULT PRIVILEGES FOR USER <database> IN SCHEMA public GRANT ALL ON TABLE
 ALTER DEFAULT PRIVILEGES FOR USER <database> IN SCHEMA public GRANT ALL ON SEQUENCES TO <username>
 ```
 
+### Listing Users
+
+#### Using the Database Dashboard
+
+1. From your web browser, open your [database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-postgresql-dashboard)
+2. Click the **Users** tab
+3. The list of available users is displayed under the **User Management**
+   section
+
+#### Using the Command Line
+
+1. Make sure you have correctly [setup the Scalingo command line tool]({% post_url platform/cli/2000-01-01-start %})
+2. From the command line, list the addons attached to your app:
+   ```bash
+   scalingo --app my-app addons
+   ```
+   The output should look like this:
+   ```text
+   +------------+-----------------------------------------+-------------------------+---------+
+   |   ADDON    |                   ID                    |          PLAN           | STATUS  |
+   +------------+-----------------------------------------+-------------------------+---------+
+   | PostgreSQL | ad-50d873b0-7760-4a4b-8d1a-760ec997b481 | postgresql-starter-1024 | running |
+   +------------+-----------------------------------------+-------------------------+---------+
+   ```
+3. Locate the `ID` of the addon for which you want to list users
+4. From the command line, run the following command to list the users:
+   ```bash
+   scalingo --app my-app --addon <addon_id> database-users-list
+   ```
+   The output should look like this:
+   ```text
+   +----------------+-----------+-----------+---------------------+
+   |    USERNAME    | READ-ONLY | PROTECTED | PASSWORD ENCRYPTION |
+   +----------------+-----------+-----------+---------------------+
+   | my_app_4553    | false     | true      | SCRAM-SHA-256       |
+   | my_app_4553_rw | false     | false     | MD5                 |
```

Comment:
Shouldn't it read `SCRAM-SHA-256` everywhere instead of `MD5`?

Reply:
Nope, the two are possible